src/App.jsx

```diff
@@ -2,20 +2,23 @@ import { Routes, Route } from "react-router-dom";
 import Header from "./components/Header";
 import ArticlesPage from "./pages/ArticlesPage";
 import ArticlePage from "./pages/ArticlePage";
+import { UserProvider } from "./contexts/User";
 
 function App() {
   return (
     <>
-      <Header />
-      <Routes>
-        <Route path="/" element={<ArticlesPage />} />
-        <Route path="/articles" element={<ArticlesPage />} />
-        <Route path="/articles/:article_id" element={<ArticlePage />} />
-        <Route
-          path="/articles/:article_id/comments"
-          element={<ArticlePage />}
-        />
-      </Routes>
+      <UserProvider>
+        <Header />
+        <Routes>
+          <Route path="/" element={<ArticlesPage />} />
+          <Route path="/articles" element={<ArticlesPage />} />
+          <Route path="/articles/:article_id" element={<ArticlePage />} />
+          <Route
+            path="/articles/:article_id/comments"
```

Comment:
do we need an additional route for this if the route above will display the same thing?